datastore/importers/nmdb-import-show-neighbor/Parser.test.cpp

```diff
@@ -0,0 +1,84 @@
+// =============================================================================
```

Comment:
Thanks for making this.  Since there are other parts you are not focusing on, I will provide additional sample cases and logic for the other parts.

Reply:
I've pushed the additional test logic for the parts not the focus of this PR.